rakefile.rb

```diff
@@ -38,32 +38,42 @@
 approval_tests = "tests/FakeItEasy.Tests.Approval/bin/Release/FakeItEasy.Tests.Approval.dll"
 
 repo = 'FakeItEasy/FakeItEasy'
+ci_server_url = 'http://teamcity.codebetter.com/admin/editBuildParams.html?id=buildType:bt929'

```

Comment:
Whilst we're here, shall we press this button

![image](https://cloud.githubusercontent.com/assets/677704/15654681/78f0153c-2696-11e6-8121-72712ad15aae.png)

to get rid of the opaque number and change it to

![image](https://cloud.githubusercontent.com/assets/677704/15654692/96482c82-2696-11e6-8b73-b4f2b7e437c1.png)

?